bin/validate-json

```diff
@@ -221,7 +221,9 @@ try {
     $validator->check($data, $schema);
 
     if ($validator->isValid()) {
-        echo "OK. The supplied JSON validates against the schema.\n";
+        if(isset($arOptions['--verbose'])) {
+            echo "OK. The supplied JSON validates against the schema.\n";
+        }
     } else {
         echo "JSON does not validate. Violations:\n";
         foreach ($validator->getErrors() as $error) {
```

Comment:
scripts or pipelines where you don't care about the error output (and don't want the clutter); you just want to validate. Lots of linux / unix scripts only care about the exit status being zero (OK) or nonzero (not OK), which this script complies with.